packages/multiselect/src/css/index.css

```diff
@@ -79,11 +79,8 @@
   margin: calc(4px / 2);
 }
 
-.psds-multi-select__input {
+.psds-multi-select__input-container input {
   min-height: 24px;
   min-width: 50px;
 }
 
-.psds-multi-select__input:disabled {
```

Comment:
This was unused afaik. Field component has this attribute on its input always.

Reply:
Yeah I prefer not to ever use a non direct selector or reference a tag. I already resolved this using a better selector see my PR